src/ecomm-content.js

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+
+export const ecommSlides = [
+    {
+        title: 'This will be the first title',
```

Comment:
Can't see this on the site anywhere so I am not sure if it's a problem, but seems like test data shouldn't be pushed live